src/azul/plugins/__init__.py

```diff
@@ -218,6 +218,10 @@ def _source_ref_cls(self) -> Type[SOURCE_REF]:
         return ref_cls
 
     def source_from_json(self, ref: JSON) -> SOURCE_REF:
+        """
+        Instance a :class:`SourceRef` from its JSON representation. The expected
```

Comment:
Looking at MerriamWebster this isn't correct use of the verb `to instance`. What this returns is neither a citation nor an example.